sdk/communication/Azure.Communication.PhoneNumbers/tests/PhoneNumbersClient/PhoneNumbersClientLiveTests.cs

```diff
@@ -197,6 +197,84 @@ public async Task UpdateCapabilitiesAsync()
             Assert.AreEqual(200, updateOperation.GetRawResponse().Status);
         }
 
+        [Test]
+        public async Task ReleaseUnauthorizedNumber()
+        {
+            var client = CreateClient();
+            try
+            {
+                var releaseOperation = await client.StartReleasePhoneNumberAsync(UnauthorizedNumber);
+            }
+            catch (RequestFailedException ex)
+            {
+                Assert.AreEqual(400, ex.Status);
+                Assert.NotNull(ex.Message);
+            }
+        }
+
+        [Test]
+        public async Task UpdateCapabilitiesUnauthorizedNumber()
+        {
+            var capabilities = new PhoneNumberCapabilities(calling: PhoneNumberCapabilityType.None, sms: PhoneNumberCapabilityType.Outbound);
+            var client = CreateClient();
+            try
+            {
+                var UpdateCapabilitiesOperation = await client.StartUpdateCapabilitiesAsync(UnauthorizedNumber);
+            }
+            catch (RequestFailedException ex)
+            {
+                Assert.AreEqual(400, ex.Status);
+                Assert.NotNull(ex.Message);
+            }
+        }
+
+        [Test]
+        public async Task GetPurchasedUnauthorizedNumber()
+        {
+            var client = CreateClient();
+            try
+            {
+                var purchaseOperation = await client.GetPurchasedPhoneNumberAsync(UnauthorizedNumber);
+            }
+            catch (Exception ex)
+            {
+                Assert.True(ex.Message.Contains("Unable to find"));
```

Comment:
I thought we decided not to look at the message content?

Reply:
removed